src/components/elementView/elementTabContainer.tsx

```diff
@@ -95,10 +95,50 @@ class ElementTabContainer extends React.Component<
             />
           }
         />
+
         <Tab
-          className="tab do-not-print"
-          id="assets"
+          className="tab tab-header do-not-print"
+          id="assets-header"
           title="Assets"
+          disabled={true}
+        />
+
+        <Tab
+          className="tab-sub do-not-print"
+          id="assets"
+          title="Racked Assets"
```

Comment:
this tab already existed -> renamed title from Assets to Racked Assets and updated css